provision/files/hosts.ci

```diff
@@ -0,0 +1,55 @@
+# Create an OSEv3 group that contains the masters and nodes groups
+[OSEv3:children]
+masters
+nodes
+etcd
+nfs_registry
+
+# Set variables common for all OSEv3 hosts
+[OSEv3:vars]
+# SSH user, this user should allow ssh based auth without requiring a password
+ansible_ssh_user=root
+
+# If ansible_ssh_user is not root, ansible_become must be set to true
+debug_level=4
+
+openshift_master_api_port=8443
+openshift_deployment_type=origin
+openshift_release=v3.9
```

Comment:
Can we maybe use templating here? This is to make sure we can run CI on different versions without too much of a hassle. Similarly for the variable `openshift_pkg_version` as well.

Reply:
This file itself is a template. Whenever we do a version update, we may need to alter things in the inventory file and the process of triggering the ansible playbook. As this wont be frequent, I believe if we keep things in single place, it would be easy.